howto/blade-id.md

```diff
@@ -30,17 +30,16 @@ Note that the Blade ID resistor is different from the in-line resistor normally
 
 The V1 electronics is still one of the best pictures for showing how this can work: [https://fredrik.hubbe.net/lightsaber/electronics.html](https://fredrik.hubbe.net/lightsaber/electronics.html)
 
-Unfortunately, it turns out that Proffieboard V1.5 and V2.2s are unable to do Blade ID in the same way as TeensySabers do, because there is no way to enable a pullup/pulldown resistor while doing an analog read from the pin. Several workarounds exist for this in ProffieOS 3.x.  
-*NOTE* - These are not needed for a Proffieboard V3.
-V3 already has what it needs internally.  
+Unfortunately, it turns out that Proffieboard V1.5 and V2.2s are unable to do Blade ID in the same way as TeensySabers do, because there is no way to enable a pullup/pulldown resistor while doing an analog read from the pin. Several options can be defined to help with this as of ProffieOS 3.x.  
+*NOTE* - These are not needed for a Proffieboard V3. V3 already has a Bridged Pullup internally.  
 
-The first option would be adding this to the config file:
+The first BLADE_ID_CLASS option is the default on Proffieboards V1.5 and V2.2s. Therefore it does not need to be specified in the config file, but here is what it is and what it does:
 
 ```cpp
 #define BLADE_ID_CLASS SnapshotBladeID<bladeIdentifyPin> 
 ```
 
-The default blade ID class (SnapshotBladeID) charges up the internal sampling capacitor, then connects the sampling capacitor to the Blade ID pin for a *very* short time, and then does the analog-to-digital conversion. This should give consistent values for the same blade, but unfortunately, the values will not reflect the value of the Blade ID resistor, so the blades array has to be configured with measured values, not with values based on the resistor in the blade. Measured values can be found by using Serial Monitor in Arduino, and entering `scanid` while the blade is off.
+The default blade ID class (SnapshotBladeID) charges up the internal sampling capacitor, then connects the sampling capacitor to the Blade ID pin for a *very* short time, and then does the analog-to-digital conversion. This should give consistent values for the same blade, but unfortunately, the values will not reflect the actual value of the Blade ID resistor in the blade. The blades array has to be configured with these measured values instead. Measured values can be found by using Serial Monitor in Arduino, and entering `id` while the blade is off. This will report "BLADE ID: N" where N is the measured value.
```

Comment:
change "The default blade ID class (SnapshotBladeID) ..." to just "SnapshotBladeID ..."

Reply:
done